subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
+#define LOG_MODULE_NAME bt_ctlr_ull_iso
+#include "common/log.h"
+#include "hal/debug.h"
+
+/* Contains vendor specific argument, function must be implmented by vendors */
+__weak uint8_t ll_configure_data_path(uint8_t data_path_direction,
```

Comment:
Similar weak functions are placed in hci.c in #29947. I leave it to other reviewers which way they prefer.

Reply:
Since this an ll function, I think it makes sense to have it here (and not hci)